docs/4.2/production.md

```diff
@@ -0,0 +1,269 @@
+   for Teleport should be done in the [configuration file](admin-guide.md#configuration).
+
+2. The **ExecReload** command allows admins to run `systemctl reload teleport`.
+   This will attempt to perform a graceful restart of _*but it only works if
+   network-based backend storage like [DynamoDB](../configuration/#storage) or
+   [etc 3.3](../configuration/#storage) is configured*_. Graceful Restarts will
+   fork a new process to handle new incoming requests and leave the old daemon
+   process running until existing clients disconnect.
+
+### Start the Teleport Service
+
+You can start Teleport as a Systemd Unit by enabling the `.service` file
+with the `systemctl` tool.
+
+```bash
+$ cd /etc/systemd/system
+# Use your text editor of choice to create the .service file
+# Here we use vim
+$ vi teleport.service
+# use the file above as is, or customize as needed
+# save the file
+$ systemctl enable teleport
+$ systemctl start teleport
+# show the status of the unit
+$ systecmtl status teleport
+# follow tail of service logs
+$ journalctl -fu teleport
+# If you modify teleport.service later you will need to
+# reload the systemctl daemon and reload teleport
+# to apply your changes
+$ systemctl daemon-reload
+$ systemctl reload teleport
+```
+
+You can also perform restarts or upgrades by sending `kill` signals
+to a Teleport daemon manually.
+
+| Signal                  | Teleport Daemon Behavior
+|-------------------------|---------------------------------------
+| `USR1`                  | Dumps diagnostics/debugging information into syslog.
+| `TERM`, `INT` or `KILL` | Immediate non-graceful shutdown. All existing connections will be dropped.
+| `USR2`                  | Forks a new Teleport daemon to serve new connections.
+| `HUP`                   | Forks a new Teleport daemon to serve new connections **and** initiates the graceful shutdown of the existing process when there are no more clients connected to it. This is the signal sent to trigger a graceful restart.
+
+### Adding Nodes to the Cluster
+
+We've written a dedicated guide on [Adding Nodes to your
+Cluster](admin-guide.md#adding-nodes-to-the-cluster) which shows how to generate or set join tokens and
+use them to add nodes.
+
+## Security Considerations
+
+### SSL/TLS for Teleport Proxy
+
+TLS stands for Transport Layer Security (TLS), and its now-deprecated predecessor,
+Secure Sockets Layer (SSL).  Teleport require TLS authentication to ensure that 
+communication between nodes, clients and web proxy remains secure and comes from 
+a trusted source.  
+
+During our [quickstart](quickstart.md) guide we skip over setting up TLS, so you can quickly try Teleport.
+Obtaining a TLS certificate is easy and is free with thank to [Let's Encrypt](https://letsencrypt.org/). If you already have a certificate these should be uploaded to the Teleport Proxy and 
+can be set via `https_key_file` and `https_cert_file`.
```

Comment:
Maybe provide an example of which cert files from LetsEncrypt you need to use? For example: if you use certbot, you get this list of files provided:

```
README
cert.pem
chain.pem
fullchain.pem
privkey.pem
```

The files that are needed for Teleport are these:

```
https_key_file: /path/to/certs/privkey.pem
https_cert_file: /path/to/certs/fullchain.pem
```